packages/kotti-checkbox/src/Checkbox.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<label class="form-checkbox">
+		<input
+			class="form-checkbox__input"
+			v-model="model"
+			:name="name"
+			type="checkbox"
+		/>
+		<i class="form-icon" />
+		<span class="form-checkbox__label">
+			<slot>{{ label }}</slot>
+		</span>
+	</label>
+</template>
+
+<script>
+export default {
+	name: 'KtCheckbox',
+	props: {
+		label: { types: [String] },
```

Comment:
```suggestion
		label: { required: true, type: String },
```

Reply:
@tomquinonero the `label` seems to be required, judging from the template. Otherwise, add `default: null` and add a `v-if="label !== null"`.